### PR TITLE
Topic/completely 2.11 probe

### PR DESCRIPTION
**Pull request by som-snytt**
See if it survives Scabot.

Scabot should be SCARbot, for the Scathing Review bot.

### CODE REVIEW

**Comment by som-snytt** [comment]
@SethTisue the last commit detunes the power mode banner. It was just too much power.

**Comment by som-snytt** [comment]
Some test output includes the "classic" power banner. Other REPL output changed line numbers or indentation. One may assume that line number and indentation changes are not what I was looking forward to this evening.

**Comment by som-snytt** [comment]
In the merge with many commits, I failed to bring the indentation fix forward. Too bad it's necessary to reapply a conflicting delta more than once.

For the line number change, I tweaked the imports code to import possible completions only when invoked for completion (i.e., from presentationCompile).

**Comment by som-snytt** [comment]
@adriaanm Yup.

**Comment by adriaanm** [comment]
Cool! All I did in my branch was rebase the completion wip on my
refactoring. This is much more complete, so I'll rebase my stuff on master
once this is in.

**Comment by som-snytt** [comment]
Reworked at https://github.com/scala/scala/pull/4659 with the recommended reductions but no additions.

File: src/repl/scala/tools/nsc/interpreter/PresentationCompilation.scala [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        override def results: List[compiler.Member] = Nil
+      }
+    }
+    def completionsOf(buf: String, cursor: Int): CompletionResult = {
+      val offset = preambleLength
+      val pos1 = unit.source.position(offset + cursor)
+      import compiler._
+      val focus1: Tree = typedTreeAt(pos1)
+      focus1 match {
+        case sel @ Select(qual, name) =>
```
Comment (MasseGuillaume) [inline src/repl/scala/tools/nsc/interpreter/PresentationCompilation.scala:80]: Missing `compiler.Import(expr, _)` ? 

https://github.com/MasseGuillaume/ScalaKata2/blob/master/eval/src/main/scala/com.scalakata/Compiler.scala#L85

**Pull request closed without merging**